docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -137,13 +140,17 @@ Emitted when the ownership is transferred.
 :RoleRevoked: xref:#IAccessControl-RoleRevoked[RoleRevoked]
 :RoleAdminChanged: xref:#IAccessControl-RoleAdminChanged[RoleAdminChanged]
 
-[.hljs-theme-light.nopadding]
 ```javascript
 use openzeppelin::access::accesscontrol::interface::IAccessControl;
 ```
 
-External interface of AccessControl declared to support {src5} detection.
-The {src5} ID for this interface is `0x23700be02858dbe2ac4dc9c9f66d0b6b0ed81ec7f970ca6844500a56ff61751`.
+External interface of AccessControl.
+
+[.contract-index]
+.{inner-src5} ID
```

Comment:
nice! but the formatting looks a bit weird tho
![image](https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/9e66e4a1-516b-47b9-abc2-88c319e2ed5c)
what about incorporating "ID" into the link?